cddl_gen/cddl_gen.py

```diff
@@ -877,9 +877,22 @@ def cbor_var_condition(self):
     def choice_var_condition(self):
         return self.type == "UNION"
 
+    def reduced_key_var_condition(self):
```

Comment:
Seems like all of these have a oneline comment explaining what they do, maybe we should add one here as well?